### PR TITLE
Add Awaitility Dependency & Import die Bibliothek

### PR DESCRIPTION
**Pull request by basselali97**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -85,7 +92,7 @@
                             <java.util.logging.manager>
                                 org.jboss.logmanager.LogManager
                             </java.util.logging.manager>
-                            <maven.home>${maven.home}</maven.home>
+                            <maven.home>${env.MAVEN_HOME}</maven.home>
```
Comment (astanik) [inline pom.xml:95]: Please, do not set the maven home in a pom.xml. The maven home is an environment variable that should be provided by the os system.

File: pom.xml [hunk 2]
```diff
@@ -114,7 +121,7 @@
                                         ${project.build.directory}/${project.build.finalName}-runner</native.image.path>
                                     <java.util.logging.manager>
                                         org.jboss.logmanager.LogManager</java.util.logging.manager>
-                                    <maven.home>${maven.home}</maven.home>
+                                    <maven.home>${env.MAVEN_HOME}</maven.home>
```
Comment (astanik) [inline pom.xml:124]: Please remove this.

File: pom.xml [hunk 3]
```diff
@@ -47,7 +47,14 @@
                 de.remsfal.service.entity.dto\**
         </sonar.cpd.exclusions>
     </properties>
-
+    <dependencies>
```
Comment (astanik) [inline pom.xml:50]: The dependency is only in the service module required. Therefore, please move this dependency to the remsfal-service/pom.xml

**Review by astanik — CHANGES_REQUESTED** [review]
This is just a preliminary review. A final review follows if the pipeline runs successfully

**Comment by basselali97** [comment]
Is it working now?

**Pull request closed without merging**